_lab/lab01.md

```diff
@@ -464,7 +476,8 @@ Keep in mind that in Python:
 * The `*` symbol is used for multiplication.  In algebra, we can write
   `1.8x` to mean `1.8` multiplied by `x`, however, this does not work
   in Python.  In Python you must write `1.8 * x` if you want to
-  multiply the variable `x` by 1.8.
+  multiply the variable `x` by 1.8 (or `1.5 * a` if you want to
+  multiply the variable `a` by 1.5, or `1.2 * three` if you want to multiply the variable `three`(which could encode any value (i.e. three = 10)) by 1.2 ...).
```

Comment:
`three` is not a descriptive variable name, and so we shouldn't encourage or give such examples.